theHarvester/__main__.py

```diff
@@ -848,7 +848,6 @@ async def handler(lst):
         print('\n[*] Virtual hosts:')
         print('------------------')
         for data in host_ip:
-            from theHarvester.discovery import bingsearch
```

Comment:
`from theHarvester.discovery import *` already exists at the top of the file.

Reply:
Some times it's needed as it does not actually work without that, if you tested it and it still works then am happy for it to be removed

Reply:
I haven't tested it out myself after making this change yet, only ran `pytest` after. I'll give it a shot tomorrow. There's still more I've got to go through it looks like.

Reply:
LMK if you got a command I can run that results in some vhosts. I tried a few stuff with keeping and removing the import, but both resulsts didn't have any vhosts to show. I can experiment with harvesting more a bit later.